### ISSUE TITLE
Doesn't work in IE11

### ISSUE THREAD

**Issue by dmitry-dedukhin** [ip-num/ip-num#22]
Probably this could be fixed by setting target=es5 in tsconfig.json?

**Comment by dadepo** [comment]
Hi @dmitry-dedukhin 
Thanks for bringing this to my attention.

Can you also help include the error message you get in ie11?

That will be appreciated.

**Comment by dadepo** [comment]
I tested it in ie11 and found out that fat arrows leads to syntax error in ie11. I will change the target to es5. This should make the v1.2.1 release which should be out in a couple of days.

**Comment by dmitry-dedukhin** [comment]
![1](https://user-images.githubusercontent.com/363471/57830290-51b5c480-77ba-11e9-9ccd-8612754ba5e9.png)
![2](https://user-images.githubusercontent.com/363471/57830291-51b5c480-77ba-11e9-8b7c-fbab5a85eee1.png)
Sorry for russain on screenshots but I've not found simple way to switch language in IE11.
In my environment also there was an error related to "class" keywords since IE does not support es6.
I've workaround this by including node_modules/ip-num in babel config.
Finally, I need to include string.prototype.repeat polyfill and all work as expected.

**Comment by dadepo** [comment]
v1.2.0 has been released with the generated JS now targeting es5. To use it in ie11, unfortunately the polyfil still need to be added :(

Trying to get things to work in ie11 is already sucking out the fun on working on the library, so I would not be spending any more energy/time on this for now. Perhaps in the future I will look into getting things to work out of the box with ie11...or perhaps not.

**Comment by dmitry-dedukhin** [comment]
Thank you very much!
We are supporting two latest versions of each browser so IE11 is still our target.
I hope this will be changed when MS released edge-over-chromium.

**Comment by dmitry-dedukhin** [comment]
I've got an error when update to 1.2.0

![22](https://user-images.githubusercontent.com/363471/57929802-ed2f5e00-78bc-11e9-85d2-2ef7c815dbed.png)

I just change version in package.json and replace import of IPv4Range to IPv4CidrRange:
```
import { IPv4 } from 'ip-num/IPv4';
import { IPv4CidrRange } from 'ip-num/IPv4CidrRange';
import { Validator } from 'ip-num/Validator';
```
![11](https://user-images.githubusercontent.com/363471/57929883-28319180-78bd-11e9-8527-f7fc1dfbc2de.png)

Seems the error caused by the fact that _super is undefined here.

**Comment by dadepo** [comment]
Dang! Just took a peek...it seems to be a case of cyclic dependency! I will look into a fix over the weekend...sad that the test suits also did not catch this :(

Thanks for letting me know!

**Comment by dadepo** [comment]
I pushed a prepatch release, which should fix this issue. 

You can Install via `npm install --save ip-num@1.2.1-0`

It would be appreciated if you can help check if indeed the issue is fixed.

If all is fine, I'll do some little cleaning up and release as soon as possible

**Comment by dmitry-dedukhin** [comment]
Hi!

I've checked 1.2.1-0 and all works just fine in chrome, firefox and ie11.
Unfortunately, I have no win10 and ms edge right now and can check it on monday.

Thank you very much for your work!

**Comment by dadepo** [comment]
Great!

No problem, the release can easily wait till after Monday.

Thanks!

**Comment by dmitry-dedukhin** [comment]
Hi!
I've check in ms edge, all is fine!

**Comment by dadepo** [comment]
Great!
I have published v1.2.1
Thanks for all the help!
